mining-buddy.lic

```diff
@@ -32,7 +32,17 @@ class MiningBuddy
   end
 
   def buy_deed_packet
-    order_item(8775, 14)
+    hometown = get_settings.hometown
```

Comment:
What do you think about moving these few lines (just the settings part) to `setup`?